### PR TITLE
Implement new Binary.Chunked mode

### PR DESCRIPTION
**Pull request by Ajanthy**
## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes
 - Ran FindSecurityBugs plugin and verified report? yes
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes

### CODE REVIEW

File: component/src/main/java/io/siddhi/extension/io/file/FileSource.java [hunk 1]
```diff
@@ -260,6 +262,13 @@
                         type = {DataType.BOOL},
                         defaultValue = "false"
                 ),
+                @Parameter(
+                        name = "buffer.size",
+                        description = "This parameter used to get the buffer size for mode binary.chunked.",
```
Comment (ramindu90) [inline component/src/main/java/io/siddhi/extension/io/file/FileSource.java]: Lets rephrase as "This parameter used to get the buffer size for binary.chunk mode.

File: pom.xml [hunk 2]
```diff
@@ -145,7 +145,7 @@
     <properties>
         <siddhi.version>5.1.13</siddhi.version>
         <siddhi.version.range>[5.0.0,6.0.0)</siddhi.version.range>
-        <wso2.transport.file.version>6.0.65</wso2.transport.file.version>
+        <wso2.transport.file.version>6.0.66-SNAPSHOT</wso2.transport.file.version>
```
Comment (ramindu90) [inline pom.xml]: Why have we used a snapshot version here?

**Review by ramindu90 — CHANGES_REQUESTED** [review]

File: component/src/main/java/io/siddhi/extension/io/file/FileSource.java (hunk 1 above)
Comment (Ajanthy) [reply component/src/main/java/io/siddhi/extension/io/file/FileSource.java]: fixed in 834102db60dbbad24ec59094d42c8f40c39d5101

File: pom.xml (hunk 2 above)
Comment (Ajanthy) [reply pom.xml]: fixed in https://github.com/siddhi-io/siddhi-io-file/pull/98/commits/4ed5c319c841473322f86c10c2198be28a85b52f

**Review by ramindu90 — APPROVED** [review]

**Pull request merged**